<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>About QxRunner</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item rowspan="2" row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>41</width>
       <height>91</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="0" column="0" >
    <widget class="QLabel" name="labelIcon" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="Resources/qxrunner.qrc" >:/icons/qxrunner_64x64.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="labelQxRunner" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text" >
      <string>QxRunner</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>51</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item rowspan="2" row="1" column="1" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" >
      <widget class="QLabel" name="labelModelNameText" >
       <property name="text" >
        <string>Model:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="labelVersionText" >
       <property name="text" >
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLabel" name="labelQtVersion" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="labelQtVersionText" >
       <property name="text" >
        <string>Qt:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="labelModelName" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>dummy</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="labelVersion" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="Resources/qxrunner.qrc" />
 </resources>
 <connections/>
</ui>
